configs/starfive/visionfive2/buildroot/kernel

```diff
@@ -0,0 +1,13 @@
+BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
+
+# branch: JH7110_VisionFive2_devel
+
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://github.com/starfive-tech/linux/tarball/JH7110_VisionFive2_devel?dummyname=JH7110_VisionFive2_devel.tar.gz"
```

Comment:
Please use the following URL format instead to match the other packages:

```
# branch: JH7110_VisionFive2_devel
# source: https://github.com/starfive-tech/linux/tree/JH7110_VisionFive2_devel
# 59cf9af678dbfa3d73f6cb86ed1ae7219da9f5c9
BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://github.com/starfive-tech/linux/archive/59cf9af678dbfa3d73f6cb86ed1ae7219da9f5c9/linux-visionfive2-5.15.0-r1.tar.gz"
```

GitHub accepts any ref in the /archive/{ref} url format (commit hash, tag, or branch) and any filename at the end of the URL.

For example:

```
https://github.com/starfive-tech/linux/archive/JH7110_VisionFive2_devel/linux-visionfive2-5.15.0-r1.tar.gz
```

Preferably we use the commit hash since it makes it reproducible.